proto/kava/committee/v1beta1/committee.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+package kava.committee.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/any.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/kava-labs/kava/x/committee/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// BaseCommittee is a common type shared by all Committees
+message BaseCommittee {
+  option (cosmos_proto.implements_interface) = "Committee";
+  option (gogoproto.goproto_stringer)        = false;
+
+  uint64         id          = 1 [(gogoproto.customname) = "ID"];
+  string         description = 2;
+  repeated bytes members     = 3 [
+    (cosmos_proto.scalar) = "cosmos.AddressBytes",
+    (gogoproto.casttype)  = "github.com/cosmos/cosmos-sdk/types.AccAddress"
+  ];
+  repeated google.protobuf.Any permissions = 4 [(cosmos_proto.accepts_interface) = "Permission"];
+
+  // Smallest percentage that must vote for a proposal to pass
+  bytes vote_threshold = 5
```

Comment:
nit: Going off what Nick said, I think it's slightly easier for clients if fields tagged with customtype for sdk Int and Dec are strings.
Looking at [what gogo does with the tag](https://github.com/gogo/protobuf/blob/master/custom_types.md), it calls the `(Un)Marshal` methods on the Int and Dec types and drops those bytes directly into the encoded proto message. The Int and Dec methods use a text marshalling func, so the bytes will always correspond to a string.
It makes it slightly clearer to clients that the `vote_threshold` field is a decimal marshalled to string, rather than some custom binary marshalling.

Reply:
make sense, I can update this